### PR TITLE
Add eigen tutorial

### PR DESCRIPTION
**Pull request by Weber-H2O2**
Some update
  - Add eigen tutorial: `eigenCall` example
  - Remove outdated code

### CODE REVIEW

File: l2/eigen-tutorials/packages/demo-eigencall/.env-sample [hunk 1]
```diff
@@ -0,0 +1,3 @@
+DEVNET_PRIVKEY="" # your private key here
+
+L2RPC="https://rinkeby.arbitrum.io/rpc" # this is rinkarby, can use any Arbitrum chain
```
Comment (eigmax) [inline l2/eigen-tutorials/packages/demo-eigencall/.env-sample:3]: Clean the comments?

File: l2/eigen-tutorials/packages/demo-eigencall/package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "name": "demo-eigencall",
+  "license": "Apache-2.0",
+  "version": "1.0.0",
+  "scripts": {
+    "build": "hardhat compile",
+    "exec": "hardhat run scripts/exec.js --network l2"
+  },
+  "devDependencies": {
+    "@nomiclabs/hardhat-ethers": "^2.0.2",
+    "arb-ts": "0.0.9",
```
Comment (eigmax) [inline l2/eigen-tutorials/packages/demo-eigencall/package.json:11]: arb-ts' version should be 0.0.13

File: l2/eigen-tutorials/packages/demo-eigencall/.env-sample (hunk 1 above)
Comment (Weber-H2O2) [reply l2/eigen-tutorials/packages/demo-eigencall/.env-sample:3]: OK.

File: l2/eigen-tutorials/packages/demo-eigencall/package.json (hunk 2 above)
Comment (Weber-H2O2) [reply l2/eigen-tutorials/packages/demo-eigencall/package.json:11]: Got it.

**Review by eigmax — APPROVED** [review]

**Pull request merged**